risc0/zkp/Cargo.toml

```diff
@@ -29,6 +29,7 @@ rustacuda_core = { version = "0.1", optional = true }
 rustacuda_derive = { version = "0.1", optional = true }
 serde = { version = "1.0", default-features = false, features = ["derive"] }
 tracing = { version = "0.1", default-features = false, features = ["attributes"] }
+blake2 = { version = "0.9.0", default-features = false }
```

Comment:
Is there any specific reason you are using an older version of the package?

Reply:
No, I just happened to have a code snippet written for an earlier version.
I have updated it to the latest and updated the relevant code.

Reply:
Great.  Fixing that also fixed the additional requirements that got added to cargo locks files